erpnext/setup/report/pending_workflow/pending_workflow.py

```
@@ -0,0 +1,192 @@
+
+@frappe.whitelist()
+def apply_workflow_action(rows, action):
+	if isinstance(rows, str):
+		rows = parse_json(rows)
+
+	for row in rows:
+		doc = frappe.get_doc(row["doctype"], row["name"])
+
+		try:
+			apply_workflow(doc, action)
+		except Exception:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)